app/views/organizations/pets/index.html.erb

```diff
@@ -14,12 +14,21 @@
   <div class="container mb-md-5 mt-md-5">
 
     <!--page heading-->
-    <div class="text-center mb-5">
-      <h1 class="section-heading text-uppercase underline">
-        <%= t('.our_pets') %>
-      </h1>
-    </div>
+    <%= content_for :header do  %>
+      <div class="text-center mb-5">
+        <h2 class="section-heading text-uppercase">
+          <%= t('.our_pets') %>
+        </h2>
+      </div>
+    <% end %>
```

Comment:
I think we should reduce the number of details in the header to be a bit smaller. This is because you'll have to repeat the div & h2 classes for every page. Instead could we do something like:
```
<%= provide(:header_title, t(".our_pets")) %>
```
and in the layout have something like:
```
      <div class="text-center mb-5">
        <h2 class="section-heading text-uppercase">
          <%= yield(:header_title) %>
        </h2>
      </div>
```

This is going to make the amount of information each page needs to write less. Let me know what you think :)